### PR TITLE
[share_plus] add macOS support

### PR DESCRIPTION
**Pull request by jpnurmi**
Using https://developer.apple.com/documentation/appkit/nssharingservicepicker

### CODE REVIEW

**Comment by jpnurmi** [comment]
share_plus_macos

File: packages/share_plus_macos/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+This project is a starting point for a Flutter
+[plug-in package](https://flutter.dev/developing-packages/),
+a specialized package that includes platform-specific implementation code for
+Android and/or iOS.
+
+For help getting started with Flutter, view our
+[online documentation](https://flutter.dev/docs), which offers tutorials,
+samples, guidance on mobile development, and a full API reference.
+
```
Comment (Schwusch) [inline packages/share_plus_macos/README.md:15]: Not that it matters a lot, but it looks a little nicer if it atleast says something along the lines of:
> Federated macos implementation package for `share_plus`

**Review by Schwusch — DISMISSED** [review]

File: packages/share_plus_macos/README.md (hunk 1 above)
Comment (jpnurmi) [reply packages/share_plus_macos/README.md]: good catch, thanks! we are using the same template for all platform plugin readme files

**Review by Schwusch — APPROVED** [review]

**Review by miquelbeltran — APPROVED** [review]
Checked the readme changes, lgtm!

**Pull request merged**